cmd/neofs-node/morph.go

```diff
@@ -211,7 +212,16 @@ func listenMorphNotifications(c *cfg) {
 		})
 	}))
 
-	setNetmapNotificationParser(c, newEpochNotification, netmapEvent.ParseNewEpoch)
+	setNetmapNotificationParser(c, newEpochNotification, func(src *state.ContainedNotificationEvent) (event.Event, error) {
+		res, err := netmapEvent.ParseNewEpoch(src)
+		if err == nil {
+			c.log.Info("new epoch event from sidechain",
```

Comment:
do we really need that to be "Info"?

Reply:
I think yes, epoch ticks is a main heartbeat of the node, and node admin should always have the ability to track the timeline. Don't you think so?

Reply:
I think epoch events are crucial for NeoFS network to function, so it makes sense to log this with `Info`.

Reply:
hm, IMO, that is more like debug. in normal situations i do not expect an admin to be informed about epoch via the logs (via the metrics or the monitoring service -- yes), only if he tries to debug smth. but ok

Reply:
on the other hand, we have the epoch metric and do not have a block metric. that could be assumed as a criterion for being smth "info"

Reply:
We still don't have strict requirement for log levels. Epoch ticks is the main thing that showa node's heartbits in logs.